accessibility/accessibility-in-pdfs.md

```diff
@@ -0,0 +1,92 @@
+# Accessibility in PDFs
+
+## A warning
+
+If you’re concerned about the accessibility of your PDF documents, then you should [consult a specialist like AbilityNet](https://www.abilitynet.org.uk/creating-accessible-pdfs) to review what you have, make expert recommendations, and potentially remediate issues for you. 
+
+## Is PDF the right choice?
+
+PDF is not the best document format for accessibility. It will always contain accessibility issues. PDF is a fixed dimension format intended for printing, and some user accessibility requirements --such as reflow, text resizing or changing text spacing-- are not supported in PDF format.
+
+We recommend that you avoid using PDF if you can, and prioritise accessible alternatives instead, such as HTML or [EPUB](https://www.w3.org/publishing/epub32/). As a reference, please read [why UK Government content should be published in HTML and not PDF](https://gds.blog.gov.uk/2018/07/16/why-gov-uk-content-should-be-published-in-html-and-not-pdf/).
+
+If you don’t have a choice, then you must purposefully build your PDFs to be accessible - **it doesn’t happen automatically**. 
+
+## Creating Accessible PDF Documents
+
+PDF is a destination format, created from a source document (e.g. Word or InDesign). In order to create accessible PDFs, the source documents themselves must be accessible. For screen reader users, this means the document must be correctly formatted with headings and alternative text, among other requirements. A correctly-formatted document can be exported as a [tagged PDF](https://helpx.adobe.com/uk/acrobat/using/creating-accessible-pdfs.html#tag_the_pdf), which allows screen reader software to describe the meaning of different pieces of text. 
```

Comment:
Probably best to include the full name of a tool the first time we mention it (e.g. "Microsoft Word", "Adobe InDesign"), then use just "Word" and "InDesign" from thereon.